build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val monocleVersion        = "3.1.0"
 lazy val scalaJavaTimeVersion  = "2.3.0"
 lazy val geminiLocalesVersion  = "0.7.0"
 lazy val jtsVersion            = "0.2.2"
-lazy val coulombVersion        = "0.6-7e4eeeb-SNAPSHOT"
+lazy val coulombVersion        = "0.6-0c31769-SNAPSHOT"
```

Comment:
@erikerlandson this works great 💯 thank you so much!

Reply:
:tada:

Reply:
@armanbilge  I'm wondering if this refactoring might fix the disambiguation of `===`